### ISSUE TITLE
Fix example for the MemberExpression option on the indent rule

### ISSUE THREAD

**Issue by royerboat** [eslint/eslint#9056]
**Tell us about your environment**

* **ESLint Version: 4.1.1**
* **Node Version: 6.9.4**
* **npm Version: 3.10.10**

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**What did you do? Please include the actual source code causing the issue.**

The docs have the following example for the MemberExpression option of the indent rule.

```js
/*eslint indent: ["error", 2, { "MemberExpression": 1 }]*/

foo
  .bar
  .baz();

// Any indentation is permitted in variable declarations and assignments.
var bip = aardvark.badger
                  .coyote;
```

However, it looks like the comment about indentation being permitted in variable declarations and assignments was removed in the following commit, but the example was not updated: https://github.com/eslint/eslint/commit/0c2a386e91308b05b4072e20b675d5ef49b9fdc1

**What did you expect to happen?**

The example should remove the comment about indentation being permitted in variable declarations and assignments since this is no longer true.

**What actually happened? Please include the actual, raw output from ESLint.**

N/A

**Comment by platinumazure** [comment]
Hi @royerboat, thanks for the issue. You are absolutely right that this needs to be fixed.

Would you like to submit a pull request?

**Comment by royerboat** [comment]
Sure, I'd be happy to.
